README.md

```diff
@@ -267,7 +267,26 @@ git push origin '<your_branch_name>'
   </td>
   </tr>
 </table>
-    
+    <hr>
+
+  ## Our Project Admins 🤓 
+
+<br>
+<table>
+<tr>
+<td align="center"><a href="https://github.com/Aaishpra"><img src="https://avatars.githubusercontent.com/u/66299533?v=4" width=150px height=150px /></a></br> <h4 style="color:white;">Shipra Verma</h4>
+
+  
+
+
+<td align="center" ><a href="https://github.com/khushishikhu"><img src="https://avatars.githubusercontent.com/u/65439761?v=4" width=150px height=150px /></a></br> <h4style="color:red;">Khushi Gautam</h4>
```

Comment:
You've missed a space here.
```
<h4 style="...">...</h4>
```
Please fix this.

Reply:
Ok sure